core/src/main/java/org/pentaho/platform/scheduler2/quartz/QuartzScheduler.java

```diff
@@ -249,7 +249,7 @@ public static MutableTrigger createQuartzTrigger( IJobTrigger jobTrigger, Quartz
     } else {
       // handle legacy imports
       startDateCal = java.util.Calendar.getInstance();
-      startDateCal.setTime( jobTrigger.getStartTime() );
```

Comment:
@peterrinehart There were no logs in the bug, so I am unsure where the NPE occurred. But if it was from "jobTrigger.getStartTime()" where "jobTrigger" is null, wouldn't this still throw an NPE?

Reply:
Good point, I'll check for that too.  That should be "impossible" but there's nothing wrong with being thorough.